src/assets/i18n/de.json

```diff
@@ -0,0 +1,159 @@
+    },
+    "export": {
+      "alertHeader": "Standortverlauf exportieren",
+      "saveMessage": "In Downloads speichern",
+      "loadingDialogMessage": "Exportiere Standortverlauf…",
+      "successfulMessage": "Standortverlauf erfolgreich exportiert",
+      "errorMessage": "Export fehlgeschlagen",
+      "permissionErrorMessage": "Export fehlgeschlagen: Berechtigungen fehlen"
+    },
+    "delete": {
+      "alertHeader": "'{{ value }}' löschen",
+      "alertMessage": "Möchtest du den Standortverlauf wirklich löschen?",
+      "loadingDialogMessage": "Lösche Standortverlauf…",
+      "successfulMessage": "Standortverlauf erfolgreich gelöscht",
+      "errorMessage": "Löschen fehlgeschlagen"
+    },
+    "explore": {
+      "title": "Erkundung",
+      "poiTimetableTitle": "Zeitplan deiner Orte",
+      "poiTimetableDescription": "Der Zeitplan visualisiert deine häufig besuchten Orte für jeden Wochentag. Wähle einen deiner besuchten Orte aus und schaue dir an, wann du dort gewesen bist.",
+      "selectPOI": "Ort auswählen",
+      "timetableChart": {
+        "hour": "Stunde"
+      }
+    },
+    "filter": {
+      "title": "Rückschlüsse filtern",
+      "byInferenceType": "Filtern nach Typ",
+      "byConfidence": "Filtern nach Wahrscheinlichkeit",
+      "low": "Gering",
+      "medium": "Mittel",
+      "high": "Hoch"
+    },
+    "insights": {
+      "title": "Rückschlüsse",
+      "noInferences": "Es konnten noch keine Rückschlüsse gezogen werden.",
+      "lookAtMapLink": "Stattdessen die Karte öffnen?"
+    },
+    "map": {
+      "title": "Karte",
+      "error": {
+        "tooManyCoordinates": "Standortverlauf kann nicht analysiert werden. Zu viele Koordinaten.",
+        "noInferencesFound": "Aus deinem Standortverlauf konnten noch keine Rückschlüsse gezogen werden.",
+        "default": "Standortverlauf kann nicht analysiert werden."
+      },
+      "loadingInferences": "Lade Rückschlüsse…",
+      "predictionButtonText": "Nächsten Aufenthaltsort bestimmen",
+      "predictionSuccess.singular": "Du wirst den markierten Ort wahrscheinlich in der nächsten Stunde besuchen",
+      "predictionSuccess.plural": "Du wirst die markierten Orte wahrscheinlich in der nächsten Stunde besuchen",
+      "predictionFail": "Dein nächster Aufenthaltsort konnte nicht bestimmt werden",
+      "toggleLocationButtonText": "Position ein- / ausblenden",
+      "createInferencesButtonText": "Rückschlüsse generieren"
+    }
+  },
+  "tracking": {
+    "title": "Standortverlauf",
+    "toggleTitle": "Aufzeichnung starten / stoppen",
+    "stateRunning": "Läuft",
+    "stateStopped": "Gestoppt",
+    "notSupported": "Auf dieser Plattform wird die Aufzeichnung des Standortes nicht unterstützt",
+    "androidBatteryHint": "Damit die App korrekt funktioniert solltest du in den Einstellungen die Batterieoptimierung für diese App deaktivieren. Weitere Informationen findest du unter <a href='https://dontkillmyapp.com'>dieser Seite</a>",
```

Comment:
there is a comma missing after "funktioniert". I would add a full stop at the end here. I would also change "findest du unter dieser Seite" to "findest du auf dieser Seite".